client/src/components/UploadProject.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from "react";
+import { Button } from "./Button";
```

Comment:
you should use export default for Components

like here :) https://github.com/DiscoDevs/DispoDisco/pull/76